packages/common/src/PublicNetworks.ts

```diff
@@ -76,9 +78,11 @@ export const PublicNetworks: PublicNetworksType = {
   },
   42161: { name: "arbitrum", etherscan: "https://arbiscan.io/" },
   43114: { name: "avalanche", etherscan: "https://snowtrace.io/" },
-  84531: { name: "base-goerli", etherscan: "https://goerli.basescan.org" },
+  8453: { name: "base", etherscan: "https://goerli.basescan.org/" },
```

Comment:
Fix Base mainnet block explorer URL.

```suggestion
  8453: { name: "base", etherscan: "https://basescan.org/" },
```